ssl_website_check.sh

```diff
@@ -44,11 +55,19 @@ function check_ssl_cert()
     printf "| %30s | %5s | %-13s | %-40s | %-50s |\n" "$host" "$port" "$end_date" "$serial" "${issuer:0:50}"
 }
 
+[ ! -e /usr/bin/openssl ] && { echo "The package openssl is not installed!"; exit; }
 
 printf "%s\n" "/--------------------------------------------------------------------------------------------------------------------------------------------------------\\"
 printf "| %30s | %5s | %-13s | %-40s | %-50s |\n" "Domain" "Port" "Expire (days)" "Serial" "Issuer"
 printf "%s\n" "|--------------------------------|-------|---------------|------------------------------------------|----------------------------------------------------|"
-for domain in "${DOMAINS[@]}"; do
-    check_ssl_cert $domain
-done
+if [[ ${DOMAINS[@]} ]]; then
```

Comment:
Any reason for the Bash special `[[` style? IMO it doesn't matter and so I'd prefer single angle brackets to keep the style as in the rest of the script.
Additionally, I think it would be nicer to rewrite this as a pre-flight check like the `openssl` check above. I.e. move the check above, before any other output, reverse the check and error out if the list is empty.